src/brain/brain_util.cpp

```diff
@@ -12,12 +12,20 @@ std::string BrainUtil::ExecutionOperatingUnitTypeToString(ExecutionOperatingUnit
       return "AGG_BUILD";
     case ExecutionOperatingUnitType::AGGREGATE_ITERATE:
       return "AGG_ITERATE";
+    case ExecutionOperatingUnitType::STATIC_AGGREGATE:
+      return "STATIC_AGGREGATE";
+    case ExecutionOperatingUnitType::HASH_JOIN:
+      return "HASH_JOIN";
+    case ExecutionOperatingUnitType::HASH_AGGREGATE:
+      return "HASH_AGGREGATE";
     case ExecutionOperatingUnitType::HASHJOIN_BUILD:
       return "HASHJOIN_BUILD";
     case ExecutionOperatingUnitType::HASHJOIN_PROBE:
       return "HASHJOIN_PROBE";
     case ExecutionOperatingUnitType::IDXJOIN:
       return "IDXJOIN";
+    case ExecutionOperatingUnitType::SORT:
+      return "SORT";
```

Comment:
Some of these types are actually redundant (I guess that's why we didn't handle them here in the first place). I'll write a separate issue about it.

Reply:
#1069